compiler/src/dotty/tools/dotc/core/quoted/PickledQuotes.scala

```diff
@@ -42,26 +55,24 @@ object PickledQuotes {
     val tastyBytes = TastyString.unpickle(expr.tasty)
     val unpickled = unpickle(tastyBytes, expr.args)
     unpickled match {
-      case PackageDef(_, (vdef: ValDef) :: Nil) => vdef.rhs
-      case PackageDef(_, (tdef: TypeDef) :: Nil) => tdef.rhs
+      case PackageDef(_, (vdef: ValDef) :: Nil) =>
+        if (vdef.name == "quote".toTermName) vdef.rhs
```

Comment:
Maybe add a dollar to the name? It's unambiguous now, but if we sometimes in the future would add toplevel ValDefs to Scala someone could write `val quote = ...`.

Reply:
It should be safe as this will only be contained in expressions `Expr[T]` and expressions do not have packages (hence not top level vals). Regardless the name with `$` is better as it shows that it is a synthetic construct.